lib/gettimeofday.c

```diff
@@ -29,7 +29,7 @@
 
 int gettimeofday(struct timeval *tv, void *tz)
 {
-#if _MSC_VER
```

Comment:
Out of curiosity, what the difference? Not all versions of _MSC_VER evaluated to 1 or true?

Reply:
#if _MSC_VER 
may lead to compile errors - maybe a standard violation for older compilers.
Nevertheless I saw this error (I could try to find the AppVeyor log, if you are really curious).
If you just want to detect if it's defined at all, use
#ifdef _MSC_VER